ui/components.d.ts

```diff
@@ -51,6 +55,8 @@ declare module 'vue' {
     NTree: typeof import('naive-ui')['NTree']
     NUpload: typeof import('naive-ui')['NUpload']
     NUploadDragger: typeof import('naive-ui')['NUploadDragger']
+    NUploadFileList: typeof import('naive-ui')['NUploadFileList']
+    NUploadTrigger: typeof import('naive-ui')['NUploadTrigger']
     NWatermark: typeof import('naive-ui')['NWatermark']
     RouterLink: typeof import('vue-router')['RouterLink']
     RouterView: typeof import('vue-router')['RouterView']
```

Comment:
There doesn't seem to be any issues with these lines of code. All the declared modules follow the recommended naming conventions, which should avoid conflicts in the project's external packages.

However, you may want to consider a few things:

- Ensure that all imports (`@@`, `declare`, etc) are placed at the top of your file instead of right before the other files' declaration.
- Use consistent indentation styles across the entire script for readability. 
- If not already done, consider renaming some symbols like `Main` and similar to something more unique if they appear frequently.

In general,
```python
#!/usr/bin/python
#coding=utf-8
import numpy as np

def create_matrix(num_rows, num_cols):
    matrix = [[1 for _ in range(num_cols)] for _ in range(num_rows)]
```
will likely look cleaner than importing the same thing twice.
  
Also, make sure you have imported necessary libraries (e.g., `numpy`) first if needed. 

If there’s anything specific that needs optimization or explanation about the code style or functionality, please provide further details so I can offer helpful insights!